### PR TITLE
added an int option for setting the seed of pytorch and numpy

### PR DESCRIPTION
**Pull request by zoidberg77**

### CODE REVIEW

**Comment by collinleiber** [comment]
I think it would be better to use the 'check_random_state' method (from sklearn.utils) for this.
We could move check_random_state from the constructor of the deep clustering methods to the set_torch_seed method.
That would be way cleaner

**Review by collinleiber — APPROVED** [review]
